CMSgov/hcgov-design-system
# [WNMGDS-1067] Authenticated header

## Summary
Updates to the header component to support the unified authenticated header

### Changed
* Moved the 'logout' link to be always present at the bottom of the menu (when in a 'logged in' state)
* The Header component will automatically append the 'Spanish' link to the bottom of the list of links when using the authenticated version. This should support the default links as well as a product wanting to use custom links
* Updated style of header to look more like the mocks

### Removed
* Support for the 'inverse' version of the header. Since the new design is the only new design, inverse is no longer needed

## How to test
1. Pull down branch. Run `nvm use`, `yarn install`, `yarn start`
2. On the local doc site, navigate to Patterns > Header
3. Play around with the logged in version. Change to Spanish and English. Confirm styles match mocks and functionality is as expected

## Still TODO
* There's something off about the translation. On the doc site, when you toggle the header into Spanish, the word for English isn't translating. This is also happening on the production site, so this is not a new bug, but a bug all the same
* ❓ **Question**: What is the "de" version of the header? Do we still need to support that? If so, does the style need to be updated?
* Check accessibility!
